client/properties.lua

```diff
@@ -0,0 +1,224 @@
+local PropertyZones = {}

```

Comment:
It stores the all of the lib.points (maybe should be named propertyPoints lol), it could be used somewhere else to access the zones (currently thinking of houserobbery that could use those zones to be able to rob these properties)